### PR TITLE
Convert BasicServiceAgreementConsumerTest to typescript

### PR DESCRIPTION
**Pull request by codehodler**

### CODE REVIEW

File: evm/v0.5/testv2/BasicServiceAgreementConsumer.test.ts [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      })
+
+      describe('when the consumer does not recognize the request ID', () => {
+        let request2: h.RunRequest
+
+        beforeEach(async () => {
+          // Create a request directly via the oracle, rather than through the
+          // chainlink client (consumer). The client should not respond to
+          // fulfillment of this request, even though the oracle will faithfully
+          // forward the fulfillment to it.
```
Comment (felder-cl) [inline evm/v0.5/testv2/BasicServiceAgreementConsumer.test.ts]: Anyway we could improve the clarity of this comment?

File: evm/v0.5/src/helpers.ts [hunk 2]
```diff
@@ -516,11 +516,39 @@ export function encodeServiceAgreement(sa: ServiceAgreement) {
   )
 }
 
-export function encodeOracleSignatures(os: OracleSignatures) {
+export function encodeOracleSignatures(os: OracleSignature) {
   const osValues = [os.vs, os.rs, os.ss]
   return ethers.utils.defaultAbiCoder.encode(ORACLE_SIGNATURES_TYPES, osValues)
 }
 
+export async function computeOracleSignature(
```
Comment (felder-cl) [inline evm/v0.5/src/helpers.ts]: Should these exported functions have comments to describe what they do?

Comment (HenryNguyen5) [reply evm/v0.5/src/helpers.ts]: Tracked here: https://www.pivotaltracker.com/story/show/170938134

**Comment by DavidCurrie1113** [comment]
Thank you for your patience . All the best I am unsubscribing.  Sent from my Bell Samsung device over Canada's largest network.
-------- Original message --------From: HenryNguyen5 <notifications@github.com> Date: 2020-01-27  5:55 p.m.  (GMT-05:00) To: smartcontractkit/chainlink <chainlink@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: Re: [smartcontractkit/chainlink] Convert BasicServiceAgreementConsumerTest to typescript (#2204) @HenryNguyen5 commented on this pull request.



In evm/v0.5/src/helpers.ts:
>    const osValues = [os.vs, os.rs, os.ss]
   return ethers.utils.defaultAbiCoder.encode(ORACLE_SIGNATURES_TYPES, osValues)
 }
 
+export async function computeOracleSignature(

Tracked here: https://www.pivotaltracker.com/story/show/170938134

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/smartcontractkit/chainlink/pull/2204?email_source=notifications\u0026email_token=AOLMROGI4A6CBUX46ZGC3SLQ75Q5FA5CNFSM4KKOR6Z2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCTG5FRA#discussion_r371528785",
"url": "https://github.com/smartcontractkit/chainlink/pull/2204?email_source=notifications\u0026email_token=AOLMROGI4A6CBUX46ZGC3SLQ75Q5FA5CNFSM4KKOR6Z2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCTG5FRA#discussion_r371528785",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Pull request merged**